### DISCUSSION TITLE
Download only audio file with fixed bitrate

### DISCUSSION THREAD

**Discussion by asteria121** [Tyrrrz/YoutubeExplode#617] [Q&A]
`
        IReadOnlyList<IStreamInfo> infoList;
        var streamManifest = await youtube.Videos.Streams.GetManifestAsync(videoID);
        var streamInfo = streamManifest
                .GetAudioOnlyStreams()
                .OrderByDescending(s => s.Container == Container.Mp4)
                .ThenByDescending(s => s.Bitrate).FirstOrDefault();

        List<IStreamInfo> infoList2 = new List<IStreamInfo>();
        infoList2 .Add(streamInfo);
        infoList = infoList2;

        var conversion = new ConversionRequestBuilder(outputPath)
                    .SetFormat("mp4")
                    .SetPreset(ConversionPreset.Medium)
                    .Build();

        await youtube.Videos.DownloadAsync(infoList, conversion);`

I want to stream it by NAudio. But if I download sound file like this code, bitrate is not constant (moves 100Kbps~150Kbps) and NAudio throws exception like "Mp3FileReader does not support samplerate changes". Is there any way to download only audio and fix bitrate like 192Kbps?

**Comment by Tyrrrz** [comment]
Are you downloading it as `mp4`? You can download it directly as `mp3`.
